Spanish translation of the new elements

Ruben0705 opened the pull request:
- Spanish translation of the new elements

With the new update, several elements have been added and others have been modified, here are the ones I have been able to find to translate them.

WishaWoshi committed:
Add ETCo2 Item

WishaWoshi committed:
Add Nasal Cannula

Remove ETCo2 Monitor, going to add this functionality to the vitals monitor

WishaWoshi committed:
Add Nasal Cannula

WishaWoshi committed:
Add ETCo2 Variables

WishaWoshi committed:
Add Breathrate variables

WishaWoshi committed:
Add ETCo2/Breathrate variables

WishaWoshi committed:
Add getETCo2 function

WishaWoshi committed:
Add CannulaActive string

WishaWoshi committed:
Add etco2/breathrate to vitals monitor

WishaWoshi committed:
Fix bug so quick view is now removed upon exiting

WishaWoshi committed:
Initial commit

apo-tle committed:
Initial commit

apo-tle committed:
Work on getETCo2 func, added pulseless and CPR states

apo-tle committed:
Update getETCo2 function to check for all potential modifiers of ETCo2 level

apo-tle committed:
Update addons/breathing/functions/fnc_getETCo2.sqf

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

apo-tle committed:
Update addons/breathing/functions/fnc_getETCo2.sqf

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

apo-tle committed:
Update addons/breathing/functions/fnc_getETCo2.sqf

Co-authored-by: Mia <95135995+MissHeda@users.noreply.github.com>

apo-tle committed:
Update addons/breathing/functions/fnc_getETCo2.sqf

Co-authored-by: Mia <95135995+MissHeda@users.noreply.github.com>

apo-tle committed:
Update addons/breathing/functions/fnc_getETCo2.sqf

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

apo-tle committed:
Merge branch 'KAT-Advanced-Medical:dev-Tomcat' into dev-Tomcat

apo-tle committed:
Add requested changes

apo-tle committed:
Move if statements to switch case

apo-tle committed:
Add random offset to ETCo2 value to simulate variance

apo-tle committed:
Add fnc_getRespiratoryRate

apo-tle committed:
Change lostBlood coefficient

apo-tle committed:
Fix stringtable

apo-tle committed:
Add requested changes

apo-tle committed:
Migrate NasalCannula classes to breathing rather than airway

apo-tle committed:
Initial setup for inserting nasal cannula

apo-tle committed:
Fix stringtable

apo-tle committed:
Adjust ETCo2 values under CPR

apo-tle committed:
Update Respiratory Rate to compensate for CPR, BVM

apo-tle committed:
Change hasCannula vars to hasEtco2Monitor (futureproofing)

apo-tle committed:
Update NasalCannula

apo-tle committed:
Fix Medical_Treatment_Actions

apo-tle committed:
Trying to fix Medical_Treatment_Actions

apo-tle committed:
Add placing/removing nasal cannula

apo-tle committed:
Add Nasal Cannula functionality

apo-tle committed:
Add Nasal Cannula to overview when inserted

apo-tle committed:
Add increasing etco2/RR for lower blood pH

apo-tle committed:
Merge branch 'KAT-Advanced-Medical:dev-Tomcat' into dev-Tomcat

apo-tle committed:
Add KingLT able to monitor etco2

apo-tle committed:
Add NasalCannula image to body gui

apo-tle committed:
Merge branch 'dev-Tomcat' of https://github.com/apo-tle/KAM into dev-Tomcat

apo-tle committed:
Add NasalCannula ui element in inventory

apo-tle committed:
Add setting for enabling etco2

apo-tle committed:
Fix formatting

apo-tle committed:
Add kPa to ETCo2 monitoring

MiszczuZPolski committed:
Merge remote-tracking branch 'upstream/dev-tomcat' into dev-Tomcat

MiszczuZPolski committed:
merger

MiszczuZPolski committed:
update spaces

Ruben0705 committed:
Spanish translation of the new elements

Ruben0705 committed:
With the new update, several elements have been
added and others have been modified, here are the ones I have been able to find to translate them

Ruben0705 committed:
Tag close

Ruben0705 committed:
Indenting

Cplhardcore committed:
Pharma - Add EACA clear trauma setting (#515)

**When merged this pull request will:**
- Adds a checkbox in settings to allow EACA to remove trauma or just
stitch wounds
- 
### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: Blue <itzblueman@gmail.com>

Cplhardcore committed:
Pharma - Fix for broken alpha action (#511)

**When merged this pull request will:**
-Fix alpha action affecting bleed rate as currently it does nothing to
bleed rate

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: Blue <itzblueman@gmail.com>

Cplhardcore committed:
Pharma - Changed Max Dosages (#514)

**When merged this pull request will:**
- With the recent ace update, overdose calculations were changed to be
variable, with a chance to suffer an overdose based on the max dose
deviation https://github.com/acemod/ACE3/pull/9746
- with this PR it will change the default deviation of 2 to more
realistic values depending on the medication, eg, fentanyl is now a 1,
so any dose over 2 will send then into OD

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

BlueTheKing committed:
Breathing - Fix PneumothoraxAlwaysVisible (#517)

**When merged this pull request will:**
- Fix PneumothoraxAlwaysVisible not working as intended

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

BlueTheKing committed:
Breathing - Fix Inspect Chest inaccuracy (#519)

**When merged this pull request will:**
- Fix Inspect Chest giving inaccurate hint if patient's head is
hyperextended

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Gilfor commented on addons/airway/stringtable.xml:
```diff
@@ -738,27 +740,31 @@
         </Key>
         <Key ID="STR_KAT_Airway_Suction_Desc_Short">
             <English>Manual Suction Pump is a medical suction device for airway suction manually. This alternative to electrical devices.</English>
+            <Spanish>La bomba de succión manual es un dispositivo de succión médico para la succión manual de las vías respiratorias. Esta alternativa a los dispositivos eléctricos.</Spanish>
```
```suggestion
            <Spanish>La bomba de succión manual es un dispositivo de succión médico para la succión manual de las vías respiratorias. Es alternativa a los dispositivos eléctricos.</Spanish>
```

BlueTheKing committed:
Breathing - Fix oxygen saturation not deteriorating from airway obstruction (#513)

**When merged this pull request will:**
- Fix `handleBreathing` exiting early if patient had no pneumothorax,
impacting unconsciousness by ignoring airway obstruction/occlusion.
### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Gilfor commented on addons/breathing/stringtable.xml:
```diff
@@ -3,12 +3,14 @@
         </Key>
         <Key ID="STR_KAT_Breathing_SETTING_ENABLE_DESC">
             <English>Sets if SPO2 simulation and according functionality like Pneumothorax and \nAirway: occlusion &amp; obstruction (SPO2 reduction will be deactivated) should be activated</English>
+            <Spanish>Establece si la simulación de SPO2 y según la funcionalidad como Neumotórax y \nVía aérea: oclusión &amp; La obstrucción (se desactivará la reducción de SPO2) debe estar activada</Spanish>
```
```suggestion
            <Spanish>Establece si la simulación de SPO2 y las funcionalidades del mismo como Neumotórax y \nvías respiratorias: oclusión/obstrucción (se desactivará la reducción de SPO2). deberían estar activadas</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_1">
             <English>Unpack Trauma Section</English>
+            <Spanish>Desenpaquetar sección de miscelaneos:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de traumas:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_1_Repack">
             <English>Repack Trauma Section</English>
+            <Spanish>Desenpaquetar sección de trauma:</Spanish>
```
```suggestion
            <Spanish>Empaquetar sección de trauma:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_8">
             <English>Unpack Miscellaneous Section</English>
+            <Spanish>Desenpaquetar sección de miscelaneos:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de miscelaneos:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_7">
             <English>Unpack Fluid Section</English>
+            <Spanish>Desenpaquetar sección de fluidos:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de fluidos:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_6">
             <English>Unpack Pharmaceutical Section</English>
+            <Spanish>Desenpaquetar sección de farmacia:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de farmacia:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_5">
             <English>Unpack Airway Section</English>
+            <Spanish>Desenpaquetar sección de vías respiratorias:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de vías respiratorias:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_4">
             <English>Unpack Breathing Section</English>
+            <Spanish>Desenpaquetar sección de respiración:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de respiración:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_3">
             <English>Unpack Pain Management Section</English>
+            <Spanish>Desenpaquetar sección de analgesicos:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de analgesicos:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_FAK_Slot_2">
             <English>Unpack Bandage Section</English>
+            <Spanish>Desenpaquetar sección de vendaje:</Spanish>
```
```suggestion
            <Spanish>Desempaquetar sección de vendaje:</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_SETTING_FAK_ItemColor">
             <English>Hint Item Color</English>
+            <Spanish>Nota color del elemento</Spanish>
```
```suggestion
            <Spanish>Color del objeto en la nota</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_SETTING_FAK_ItemColor_DESC">
             <English>The item color of the text that is displayed via a hint</English>
+            <Spanish>El color del elemento del texto que se muestra mediante una nota.</Spanish>
```
```suggestion
            <Spanish>El color del elemento que aparece en el texto de la nota.</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1241,195 +1251,218 @@
         </Key>
         <Key ID="STR_KAT_Misc_SETTING_FAK_SlotColor">
             <English>Hint Section Color</English>
+            <Spanish>Seccióm de color del la nota</Spanish>
```
```suggestion
            <Spanish>Sección de color de la nota</Spanish>
```

Gilfor commented on addons/misc/stringtable.xml:
```diff
@@ -1471,13 +1504,16 @@
         </Key>
         <Key ID="STR_KAT_Misc_Use_SurgicalKitFullBody">
             <English>Use Surgical Kit (Full Body)</English>
+            <Spanish>Utilice kit quirúrgico (cuerpo completo)</Spanish>
```
```suggestion
            <Spanish>Utilizar kit quirúrgico (cuerpo completo)</Spanish>
```

Gilfor reviewed (changes requested):
En misc de la 1244 a la 1255 borras líneas, ponlas de nuevo

Buen curro, gracias.

Ruben0705 committed:
Update addons/airway/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/breathing/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

Ruben0705 committed:
Update addons/misc/stringtable.xml

Co-authored-by: Gilfor <gaizkamaxular@gmail.com>

BlueTheKing committed:
Airway/Breathing/Circulation - Fix 0% chance (#520)

**When merged this pull request will:**
- Prevent random rolls from being able to succeed when set to 0% in
settings

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Gilfor commented:
En misc de la 1244 a la 1255 borras líneas, ponlas de nuevo

Tienes que hacer esto para que te lo hagan merge.

Ruben0705 commented:
Lo estoy intentando pero puede ser que tenga que borrar la branch porque no me deja hacer el commit y tampoco puedo hacerle un pull de lo que te comitee, me dice que es invalido porque esta branch no esta mergeada.

Ruben0705 committed:
Update stringtable.xml

Add lines 1244-1255

MiszczuZPolski committed:
Prepare update 2.14.2

mazinskihenry committed:
Airway/Breathing/Circulation/Misc/Pharma/Surgery - Add Ultrasound and REBOA Unit + H&Ts + Opioid Relief and Effectivness (#452)

**When merged this pull request will:**

- Adding a handheld ultrasound unit to immediately diagnose cardiac and
thoracic conditions
- Adding a simple REBOA unit to stop bleeding from lower extremities and
chest
- Adding H&T conditions to drop patients into cardiac arrest and better
track those conditions from a single variable

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

MiszczuZPolski committed:
Prepare dev update 2.15.0

Apricot-ale committed:
Add Japanese (#1)

Cplhardcore committed:
Added Max Dose Deviation to morphine and phenylphrine autoinjector

apo-tle committed:
Fix pH offset if pharma is not loaded

apo-tle committed:
Add nasal cannula to medical crate

apo-tle committed:
Merge pull request #2 from apo-tle/dev-tomcat

Fix pH offset if pharma not loaded

fijik124 committed:
Breathing, Pharma - Add model for AAT Kit and fix IV needle model (#524)

**When merged this pull request will:**
- Title

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: fijik1124 <business.galaxyside.eu@gmail.com>
Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

V1nsyara committed:
Translate - Russian for 2.15 (#521)

**When merged this pull request will:**
- _Describe what this pull request will do_
- _Each change in a separate line_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

MiszczuZPolski committed:
Chemical - Fix mortar issues (#458)

**When merged this pull request will:**
-  Fix wrong inheritance
- Fix "No matching carry mag" that was blocking unloading 8rnd and 5rnd
ammo from mortar

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Ruben0705 committed:
Russian in misc

Ruben0705 committed:
Update stringtable.xml

Ruben0705 committed:
Update stringtable.xml

Ruben0705 committed:
Update stringtable.xml

Ruben0705 committed:
Update stringtable.xml

BlueTheKing committed:
Adjust surgery

BlueTheKing committed:
Asystole when dead

BlueTheKing committed:
&&

BlueTheKing committed:
Fix internal bleeding not clearing

regiregi22 committed:
Traslation to Spanish

Traslation to Spanish

mazinskihenry committed:
Airway/Breathing/Circulation - Add ETCo2 Monitor (#495)

**When merged this pull request will:**
- Add ETCo2 level to Vitals Monitor. ETCo2 level can be used to diagnose
respiratory events such as airway blockages and PTX deterioration, as
well as cardiac arrest
- Add Respiratory Rate to Vitals Monitor. Respiratory Rate can be used
in conjunction to ETCo2 level to diagnose problems with a patient's
airway.
- ETCo2 level and Respiratory Rate have no effect on any stats behind
resuscitating a patient, instead serving as indicators to allow doctors
to spot issues more easily.
- Add Nasal Cannula. Nasal Cannula will be required to be attached to a
patient's head in order for ETCo2 and Respiratory Rate readouts to be
visible on the Vitals Monitor. You will not be able to use the Nasal
Cannula to provide a patient with oxygen.
### To Do
- [x] Implement Respiratory Rate
- [x] Implement Nasal Cannula
- [x] Remove ETCo2/RR readouts when Nasal Cannula is not connected
- [x] Add setting to allow ETCo2/RR readouts to be displayed
- [x] Remove debugging code and tidy up

### IMPORTANT

- [x] [Development
Guidelines](https://ace3.acemod.org/wiki/development/) are read,
understood and applied.
- [x] Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Surgery - Fix Unconsciousness Required for Surgery setting (#529)

**When merged this pull request will:**
- Make Etomidate a requirement with the `Unconsciousness Required for
Surgery` setting
- Set `Surgery Causes Unconsciousness` as default

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Component - Add Traslation to Spanish (#532)

Traslation to Spanish

**When merged this pull request will:**
Traslate to Spanish

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Circulation - Fix internal bleeding persisting after respawn (#531)

**When merged this pull request will:**
- Make sure internal bleeding is cleared on full heal

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Pharma - Add Max Dose Deviation to morphine and phenylephrine autoinjector (#522)

**When merged this pull request will:**
- Add Max Dose Deviation (from
https://github.com/KAT-Advanced-Medical/KAM/commit/4bb6028d04faaece860b3db224a663c712fc083d)
to morphine and the new phenylephrine autoinjector

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Circulation - Fix AED monitor not displaying correct rhythm when dead (#530)

**When merged this pull request will:**
- Make the AED display asystole for dead patients

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Cplhardcore committed:
Pharma - Fix for mislabeled meds (#535)

**When merged this pull request will:**
- Make atropine inherit from EACA like all other meds
### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

apo-tle committed:
Airway/Breathing/Circulation - Fix EtCO2 monitor error when AED attached (#538)

**When merged this pull request will:**
- Fixes error caused when attaching AED pads to a patient.

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

MiszczuZPolski committed:
Revert "Pharma - Fix for broken alpha action" (#539)

Reverts KAT-Advanced-Medical/KAM#511

BrettMayson committed:
Pharma - Modify ace_medical_status_getBloodLoss for alpha action (#512)

**When merged this pull request will:**
- Modifies the event `ace_medical_status_getBloodLoss` to apply alpha
action factor
- Fixes a reference to alphaAction in circulation
- [ ] Requires https://github.com/acemod/ACE3/pull/9926 to be released
- [ ] Requires #511 to be reverted

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

V1nsyara committed:
Translations - Update Russian (#536)

Update:

- breathing
- circulation

Fanrod-Bones committed:
Translations - Add French translation (#537)

Update of the french translations of all modules

**When merged this pull request will:**
- Add all the missing french translations
- Make mister Frogman happy (I hope)

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

Apricot-ale committed:
Translation - Improve Japanese (for 2.15) (#509)

**When merged this pull request will:**
- _title_
- same as https://github.com/Belhun/KAM-ReaddFullStich/pull/4 he never
merged...
- same as https://github.com/mazinskihenry/KAM/pull/4 that never merged
too.
- add "EACA clear trauma" localization

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

Gilfor commented:
He estado muy liado, cualquier cosa háblame por DC a Gilfor#4767

BlueTheKing committed:
Misc/GUI - Fix error on bandaging (#543)

**When merged this pull request will:**
- Fix error after successful bandage use

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

MiszczuZPolski committed:
Pharma - Fix coagulation bug (#544)

**When merged this pull request will:**
- After respawn coag would stop working because default value was used
as array

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Fanrod-Bones committed:
Translation - Improve French (not the people) (#547)

**When merged this pull request will:**
- Add missing translations of Circulation
- Make a bad joke in the title

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

MiszczuZPolski committed:
Pharma - Adjust magazine items to use ace system (#549)

**When merged this pull request will:**
- Because of "registerItemReplacement" items added via arsenal couldnt
be removed
- Its been a while since ace update containing our feature but it seems
we didnt update it properly

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

Apricot-ale committed:
Pharma - Change KAM Painkillers to CWMP (#548)

**When merged this pull request will:**
- _Fix https://github.com/KAT-Advanced-Medical/KAM/issues/546 problem._
- _Rename whole of (KAT) Painkillers thing (except item class name) to
CWMP (from [TCCC
14.3](https://tccc.org.ua/en/guide/combat-wound-medication-pill-pack))
and item stringtable to Combat-Pill-Pack (from [DD 1380 TCCC CASUALTY
CARD](https://learning-media.allogy.com/api/v1/pdf/eb6c36b5-edbf-4e68-ae47-7a6e6536c6e7/contents))_
- _Max "ammo" count is now 10->4, weight is 1->0.5 (Guide need to
change)_

New icon;

![icon_painkillers](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/41a0e77c-ce68-4724-8526-ec888f114802)
New icon with higher resolutions (you know where it come from...)
when 64x64, almost text are can't read and texture not detailed so it's
will be ok

![icon_painkillers_256](https://github.com/KAT-Advanced-Medical/KAM/assets/50947830/81af1dba-0d95-4ed9-a42c-a687f5e8f2c7)



### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

YetheSamartaka committed:
Chemical/Misc/Surgery/Pharma - Fix Hashmap retrieval, inheritance and CWMP issues (#554)

**When merged this pull request will:**
- _Fix errors due to ACE inheritance changes where the game would not
even launch_
- _Fix error in inheritace for quadbike stretcher position points_
- _Fix error in medical menu due to ACE Hashmap change for fnc_treatment
that we have modified_
- _Fix wrong CSTRING for CWMP_
- _Fix wrong CWMP callback success referencing old Painkillers fnc_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

mazinskihenry committed:
Revert "Chemical/Misc/Surgery/Pharma - Fix Hashmap retrieval, inheritance and CWMP issues" (#555)

Cplhardcore committed:
Misc - Fix IFAK unpacking causing crash (#556)

**When merged this pull request will:**
-Fix the issue with people crashing when an FAK is unpacked when there
is not enough inventory space

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

YetheSamartaka committed:
Chemical/Misc/Surgery/Pharma - Fix Hashmap retrieval, inheritance and CWMP issues (#557)

**When merged this pull request will:**
- _Fix errors due to ACE inheritance changes where the game would not
even launch_
- _Fix error in inheritace for quadbike stretcher position points_
- _Fix error in medical menu due to ACE Hashmap change for fnc_treatment
that we have modified_
- _Fix wrong CSTRING for CWMP_
- _Fix wrong CWMP callback success referencing old Painkillers fnc_

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/)
are read, understood and applied.
- Title of this PR uses our standard template `Component -
Add|Fix|Improve|Change|Make|Remove {changes}`.

---------

Co-authored-by: MiszczuZPolski <71414303+MiszczuZPolski@users.noreply.github.com>

MiszczuZPolski committed:
Merge branch 'pr/516' into dev-Tomcat
